enhancements/sig-architecture/70-spread-policy/README.md

```diff
@@ -0,0 +1,301 @@
+}
+
+// TopologyWeightTerm defines a terminology for the weight of topologies
+type TopologyWeightTerm struct {
+	// Weight defines the relative number of clusters to schedule in this topology.
+	// +required
+	Weight int32 `json:"weight"`
+
+	// Operator represents a key's relationship to a set of values.
+	// Valid operators are In, NotIn, Exists and DoesNotExist.
+	// +required
+	Operator metav1.LabelSelectorOperator `json:"operator"`
+
+	// Values is an array of string values. If the operator is In or NotIn,
+	// the values array must be non-empty. If the operator is Exists or DoesNotExist,
+	// the values array must be empty.
+	Values []string `json:"values,omitempty"`
+}
+```
+
+### API Example
+
+#### Example 1: Joint use of a Even constraint and Affinity constraints
+```yaml
+apiVersion: cluster.open-cluster-management.io/v1alpha1
+kind: Placement
+metadata:
+  name: demo-placement
+spec:
+  # numberOfClusters is required if spreadConstraints contains
+  # a constraint of Even type.
+  numberOfClusters: 10
+  # Prediates work before the spread constraints to filter
+  # out some undesired topologies.
+  predicates:
+  - # ...
+  # SpreadConstraints contain two constraints, which work
+  # independently and jointly.
+  spreadConstraints:
+    # order is omitted for this Even constraint as there is only
+    # one of it.
+  - type: Even
+    topologyKey: provider
+    topologyKeyType: Label
+    maxSkew: 2
+    # Affininy constraints do not have a maxSkew field.
+  - type: Affinity
+    topologyKey: region
+    topologyKeyType: Label
+    # The cluster affinity api doesn't use LabelSelector
+    # directly in order to force users to organize the
+    # affinity rules of the same topologyKey together,
+    # which may be more clear.
+    topologyWeights:
+    # The weight here indicates the relative score that the clusters
+    # get when meeting the expression from the prioritizer for Affinity
+    # constraints.
+    - weight: 20
+      operator: In
+      values:
+      - us-east-1
+      - cn-hangzhou
+    - weight: 10
+      operator: In
+      values:
+      - us-west-1
+    - weight: 20
+      # NotIn expresses anti-affinity.
+      operator: NotIn
+      values:
+      - cn-shenzhen
+  - type: Affinity
+    topologyKey: zone
+    topologyKeyType: Label
+    topologyWeights:
+    - # ...
+  # I plan not to exclude the Affinity and Even constraints in Additive mode.
+  # In Exact mode, users need to specify the weight of them manually. Affinity
+  # constraints only work when there is a ClusterAffinitySpread score coordinate
+  # in prioritizerPolicy. Even constraints only work when there is a ClusterEvenSpread
+  # coordinate in prioritizerPolicy.
+  prioritizerPolicy:
+    mode: Exact
+    configurations:
+      - scoreCoordinate:
+          builtIn: ResourceAllocatableMemory
+      - scoreCoordinate:
+          builtIn: ClusterAffinitySpread
+        weight: 2
+      - scoreCoordinate:
+          builtIn: ClusterEvenSpread
```

Comment:
hrm, if this is in the prioritizer, what is the relationship of this and the weight in topologyWeight

Reply:
`topologyWeight` is used for cluster affinity which has been excluded from this proposal.